tensorboard/util/grpc_util_test.py

```diff
@@ -129,8 +129,8 @@ def handler(request, context):
 
   def test_call_with_retries_includes_version_metadata(self):
     def digest(s):
-      """Hashes a string into a 32-bit integer."""
-      return int(hashlib.sha256(s.encode("utf-8")).hexdigest(), 16) & 0xffffffff
+      """Hashes a string into a positive 32-bit signed integer."""
```

Comment:
> positive

Technically, non-negative, but I don’t have much ground for nits on
changes that fix bugs that I introduced. :-) Thanks!

Reply:
Haha yes, so sloppy of me ;)  At some point in the next two billion versions of TensorBoard, this docstring will be briefly wrong. <g>